core/src/test/java/com/spotify/missinglink/ArtifactLoaderTest.java

```diff
@@ -182,6 +182,15 @@ public void testLoadAsmJar() throws Exception {
     assertThat(artifact.classes()).hasSize(25);
   }
 
+  @Test
+  public void testLoadBouncyCastleJar() throws Exception {
+    final Artifact artifact =
+            loader.load(FilePathHelper.getPath("src/test/resources/bcprov-jdk15on-1.68.jar"));
+
+    // bcprov-jdk15on-1.68.jar is known to have 3607 classes in it when run on JVM 11
+    assertThat(artifact.classes()).hasSizeGreaterThan(1);
```

Comment:
just wondering - why not assert it has the exact amount stated in the comment, instead of > 1?

Reply:
Because that depends on the version of JVM you run this test on :smiley:

Reply:
Would be great to add this as a comment, if someone else wondered 👍

Reply:
You guys are right... the test was unnecessarily sloppy :wink: I've improved it.